terraform/040-id-broker/task-definition.json

```diff
@@ -179,20 +179,32 @@
         "value": "${mfa_totp_apisecret}"
       },
       {
-        "name": "MFA_U2F_apiBaseUrl",
-        "value": "${mfa_u2f_apibaseurl}"
+        "name": "MFA_WEBAUTHN_apiBaseUrl",
```

Comment:
Just curious as to why the mix of snake and camel cases.

Reply:
the underlying app loads it's env vars based on a prefix. so MFA_WEBAUTHN is the prefix and it loads them all into an array, so in the app this would just be referenced as `['apiBaseUrl']`

Reply:
So that it can mostly be named following the convention of env. vars (SNAKE_UPPER_CASE), but when we use `Env::getArrayFromPrefix('MFA_WEBAUTHN_')` it will return an array matching the property names of whatever component we're configuring (e.g. `['apiBaseUrl' => '...', etc.]`).